scripts/upgrade

```diff
@@ -24,37 +24,31 @@ upgrade_type=$(ynh_check_app_version_changed)
 ynh_script_progression --message="Ensuring downward compatibility..." --weight=1
 
 # If secret_key doesn't exist, create it
-if [[ -z "${secret_key:-}" ]]; then
+if [ -z "${secret_key:-}" ]; then
 	secret_key=$(ynh_string_random --length=50)
 	ynh_app_setting_set --app=$app --key=secret_key --value=$secret_key
 fi
 
-#=================================================
-# BACKUP SQLITE DATABASE AND DATAS
-#=================================================
-ynh_script_progression --message="Backup of the DB in temporary file...." --weight=5
-
-# copy data directory backup
-tmp_data_backup=$(mktemp -d)
-rsync -a $install_dir/data/ $tmp_data_backup/
-
-if [[ -f "$install_dir/config.local.user.php" ]]; then
-	cp -a $install_dir/config.local.user.php $tmp_data_backup/
-fi
-
-if  [[ $(grep "Garradin" "$tmp_data_backup/config.local.user.php") ]] ; then
-	ynh_replace_string --match_string="namespace Garradin;" --replace_string="namespace Paheko;" --target_file="$tmp_data_backup/config.local.user.php"
-fi
-
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
 
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=5
+
+	if [ ! -d "$data_dir/data" ]; then
```

Comment:
```suggestion
	if [ ! -d "$install_dir/data" ]; then
```

Reply:
Does this condition will securely remove then the files `$install_dir/config.local.yunohost.php` and `$install_dir/data/config.local.user.php` after upgrade ?